### ISSUE TITLE
Tabs: add example of local navigational tabs

### ISSUE THREAD

**Issue by andybraren** [patternfly/patternfly-next#1328]
The documentation for the current Tabs implementation reads:

> Tabs should only be used to change content views within a page. The similar-looking but semantically different Horizontal Nav component is available for general navigation use cases.

As a result, we only include examples of `.pf-c-tabs` paired with a `<div>`:

    <div class="pf-c-tabs">

In #1127 and [React #1144](https://github.com/patternfly/patternfly-react/pull/1144#issuecomment-456758118) @jgiardino explains that when a set of tabs is totally unique on the page they should be within a `<nav>` instead so that screen reader users can find them by browsing for navigational landmarks. The HTML would then be:

    <nav class="pf-c-tabs" aria-label="Local">

The Tabs component should include a new example of this variant that explains the accessibility benefits and how it should be used.

**Comment by rachael-phillips** [comment]
Can I get an update on where this issue is at?

**Comment by andybraren** [comment]
Closing this since #1303 included accessibility documentation that sufficiently covers this possibility. We're in agreement that adding a separate Tabs example isn't necessary. FYI @rachael-phillips.
